Sources/SwiftMetricsDash/SwiftMetricsDash.swift

```diff
@@ -166,40 +164,34 @@ class SwiftMetricsService: WebSocketService {
              }
         }
 
+        let envArray = [["Parameter":"Command Line","Value":"\(commandLine)"],
+                    ["Parameter":"Hostname","Value":"\(hostname)"],
+                    ["Parameter":"Number of Processors","Value":"\(numPar)"],
+                    ["Parameter":"OS Architecture","Value":"\(os)"]]
 
-        let envLine = JSON(["topic":"env","payload":[
-                ["Parameter":"Command Line","Value":"\(commandLine)"],
-                ["Parameter":"Hostname","Value":"\(hostname)"],
-                ["Parameter":"Number of Processors","Value":"\(numPar)"],
-                ["Parameter":"OS Architecture","Value":"\(os)"]
-                ]])
+        let envDashData = DashData(topic: "env", payload: envArray)
+        let data = try! encoder.encode(envDashData)
 
         for (_,connection) in connections {
-            if let messageToSend = envLine.rawString() {
-                connection.send(message: messageToSend)
-            }
+            connection.send(message: String(data: data, encoding: .utf8)!)
         }
     }
 
 
     public func sendTitle()  {
-        let titleLine = JSON(["topic":"title","payload":[
-            "title":"Application Metrics for Swift",
-            "docs": "http://github.com/RuntimeTools/SwiftMetrics"]])
+        let titleLine = "{\"topic\":\"title\",\"payload\":{\"title\":\"Application Metrics for Swift\",\"docs\": \"http://github.com/RuntimeTools/SwiftMetrics\"}}"
 
          for (_,connection) in connections {
-            if let messageToSend = titleLine.rawString() {
-                connection.send(message: messageToSend)
-            }
-        }
+             connection.send(message: titleLine)
+         }
     }
 
     public func storeHTTP(myhttp: HTTPData) {
         let localmyhttp = myhttp
     	  httpQueue.sync {
             if self.httpAggregateData.total == 0 {
                 self.httpAggregateData.total = 1
-                self.httpAggregateData.timeOfRequest = localmyhttp.timeOfRequest
+                self.httpAggregateData.time = localmyhttp.timeOfRequest
```

Comment:
should this not be 
```swift
self.httpAggregateData.time = localmyhttp.time
```

Reply:
I think `timeOfRequest` is right - it's an HTTPData - 
https://github.com/RuntimeTools/SwiftMetrics/blob/swift4/Sources/SwiftMetricsKitura/SwiftMetricsKitura.swift#L23